### ISSUE TITLE
[Suggestion] Stop Deleting All Quest POI Info

### ISSUE THREAD

**Issue by StraysFromPath** [ZhengPeiRu21/mod-individual-progression#274]
Since 2023 you can choose to disable quest POI using a config option in worldserver.

This module deletes all POI data and provides a "fix POI data" which only does 80% of the job and leaves hundreds of quest POIs broken.

Given the entire feature can be done in the config by changing a 0 to a 1 there is no reason to delete POI data and provide broken "re-enable quest marker" sqls that work but don't fully work.

This griefs me every time I update my server. :(

**Comment by SuzumiK** [comment]
Couldn't you just *not* run the `optional_remove_quest_markers.sql` or am I missing something here?

**Comment by StraysFromPath** [comment]
> Couldn't you just _not_ run the `optional_remove_quest_markers.sql` or am I missing something here?

The prepackaged / repacked version has "optional remove quest markers" pre-ran, deleting all POI data.

The "fix poi data" SQL doesn't repair all deleted data.

The entire purpose of the "remove quest markers" can be achieved in a config file.

So it's a 100% unneccesary downgrade/pointless destruction of valuable data that can't be reversed for most users.

**Comment by ZhengPeiRu21** [comment]
I agree that I should update the preview build to take advantage of this new option. It wasn't available when I first started creating the builds and I've never properly updated it.
Note that the config option doesn't remove some of the other quest tracking related changes, such as indicators for mobs dropping quest items. Even after the update, the optional SQL file can still be used to remove these changes (just as with POI markers, the quests were originally designed to be completed without this feature; it is a nice QOL improvement but trivializes cases where the player was meant to figure things out.)

I'll update the preview build at some point in the future to fix this. With that said, I do like to encourage people making customizations to consider making their own builds with the module installed rather than using the preview build - you will have tighter control and less headaches when it comes to updates. The preview build is designed for those who just want something that is "pick up and play."
